lib/importproject.cpp

```diff
@@ -150,6 +150,7 @@ static bool simplifyPathWithVariables(std::string &s, std::map<std::string, std:
 void ImportProject::FileSettings::setIncludePaths(const std::string &basepath, const std::list<std::string> &in, std::map<std::string, std::string, cppcheck::stricmp> &variables)
 {
     std::set<std::string> found;
+    // NOLINTNEXTLINE(performance-unnecessary-copy-initialization)
     const std::list<std::string> copyIn(in);
```

Comment:
@danmar Why do you need to copy this?

Reply:
to allow that `includePaths` is passed as `in` parameter

Reply:
Wait...what? That took me quite a while to understand... We should not do that but let's tackle that at a later date.